### PR TITLE
Adding manual global seed to eval

### PR DESCRIPTION
**Pull request by wshainin**
Adding CLI option for setting global seed in interactive eval script
Adding a utils folder in traiderdaive.
Adding a function to set the global seed in python, numpy, and torch.
Minor style and typing fixes

### CODE REVIEW

File: traiderdaive/utils/set_seeds.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import numpy as np
+import torch
+
+
+def set_seeds(seed: int | None = None):
+    seed = seed if seed is not None else np.random.randint(1, 99999)
+    random.seed(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+    print(f"Global Seed: {seed}")
```
Comment (dpaiton) [inline traiderdaive/utils/set_seeds.py:12]: I think we use logging elsewhere, might as well do it here.

**Review by dpaiton — APPROVED** [review]

File: traiderdaive/utils/set_seeds.py (hunk 1 above)
Comment (wshainin) [reply traiderdaive/utils/set_seeds.py:12]: haha, I initially was using logging, but ray internally sets up the logger during config.build(), so if I call set_seeds before config.build(), it doesn't get outputted, and if I call set_seeds after that, the seed doesn't actually work.
